pkg/kv/kvclient/kvcoord/dist_sender_rangefeed.go

```diff
@@ -280,6 +282,14 @@ func (a *activeRangeFeed) onRangeEvent(
 	a.RangeID = rangeID
 }
 
+func (a *activeRangeFeed) setError(err error) {
+	a.Lock()
+	defer a.Unlock()
+	a.LastErr = errors.Wrapf(err, "disconnect at %s: checkpoint %s/-%s",
+		timeutil.Now().Format(time.RFC3339), a.Resolved, timeutil.Since(a.Resolved.GoTime()))
+	a.NumErrs++
```

Comment:
is this also reaching some metric?

Reply:
(Not suggesting you add one here but it would probably be useful?)

Reply:
Yeah; it would be useful; not going to add it now because I think we need to add per-error type metrics instead. This can happen as a follow in PR